EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml

```diff
@@ -2,7 +2,7 @@
 <text xmlns="http://www.tei-c.org/ns/1.0" xml:base="https://betamasaheft.eu/">
    <body>
       <div type="edition" xml:id="Transkribus" subtype="transkribus">
-         <div type="textpart" subtype="folio" n="1">
+         <div type="textpart" subtype="image" n="1">
```

Comment:
same for all the other folia

Reply:
I have corrected the style sheet now - I think these I did manually, and wasn't very careful!! Apologies

Reply:
Do you want to manually replace all occurences of subtype="folio" to subtype="image" in this and other files?

Reply:
see line 14, 25, etc

Reply:
Yes, I have done that now

Reply:
not for this file though